app/src/routes/home.tsx

```diff
@@ -12,39 +12,45 @@ import DishCard from "../widgets/dishcard";
 import '../styles/index.css'
 import axios from "axios";
 import { Box, AppBar, Typography, Link as LinkMUI } from "@mui/material";
+import Cookies from 'js-cookie';
+
+
 
 
 const DishLog = ({dishes}) => {
+  let token = SessionToken()
   return (
     <div id="dish-log" className="mt-3">
       { dishes.map(dish => {
-        return <DishCard dish={dish} />
+        if (dish.returned.timestamp == null){
+          return <DishCard dish={dish} token={token} key={dish.id} />
+        }
       }) }
-      <div>
+      {/* <div style={{position:'fixed', left:'5px', bottom:'5px'}}>
         <button id="prev">Prev</button>
         <input type="number" id="page-number" />
         <button id="next">Next</button>
-      </div>
+      </div> */}
     </div>
   )
 };
 
-const NewUser = (user) => {
-  const content = GetDishes(user)
+const NewUser = (dishesUsed) => {
+  const content = GetDishes(dishesUsed)
```

Comment:
would love to see types on more function arguments!

Reply:
sorry kira, what do you mean by this?